src/main/java/frc/robot/utils/DeltaAngleSpeedCalcUtil.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants;
+
+public class DeltaAngleSpeedCalcUtil {
+  private static final double SPEAKER_HEIGHT_INCHES = Units.inchesToMeters(77.9528);
+  private static final double SHOOTER_SPEED_MPS = Constants.PLACEHOLDER_DOUBLE;
```

Comment:
We're going to have to measure this at some point. For now can you set it to like 10 or something?

Also it won't be possible to unit test without setting this to some non-zero value.

Can you make sure your unit test still passes if we change this number? Like maybe make this public and non-final, and set it in your unit test. Or make this variable and the below function non-static, and set this variable in the constructor. You've got different options.

Reply:
Alright